src/inputs/ToggleButton.test.tsx

```diff
@@ -0,0 +1,104 @@
+import { click, render, wait } from "src/utils/rtl";
+import { ToggleButton } from "./ToggleButton";
+
+describe("ToggleButton", () => {
+  it("renders", async () => {
+    // Given a ToggleButton
+    const r = await render(
+      <ToggleButton
+        label={"Toggle button"}
+        onClick={() => {}}
+      />
+    );
```

Comment:
**nit:** Can this be done in a single line? I personally find the code easier to scan and read through when we try to limit the number of lines of code.
```
const r = await render(<ToggleButton label={"Toggle button"} onClick={() => {}} />);
```

Reply:
Sure, I'll change them :)